boskos/ranch/ranch_test.go

```diff
@@ -1151,6 +1151,10 @@ func TestSyncResources(t *testing.T) {
 			t.Errorf("Test %v: \n got \t\t%v, \n expected \t%v", tc.name, resources, tc.expectedRes)
 		}
 		lfs, err := c.Storage.GetDynamicResourceLifeCycles()
+		if err != nil {
+			t.Errorf("failed to get dynamic resources life cycles")
```

Comment:
Please also include err in the error message

Reply:
I have done